File: DSA Learning Series/Contest-1/ReverseTheNumber.java
import java.util.Scanner;

class ReverseNumber{
    
    public static void main(String args[])
    {
        Scanner inp=new Scanner(System.in);
        int t=inp.nextInt();
        
        while(t-->0)
        {
            int n=inp.nextInt();
            int result=0;
            while(n>0)
            {
                int rem=(n%10);
                result=(result*10)+rem;
                n=n/10;
            }
            
             System.out.println(result);
        }
        
        
        
       
        
        
    }
    
    
}

File: DSA Learning Series/Contest-1/ReverseTheNumber.kt
import java.util.Scanner
fun main(args:Array<String>)
{
    val inp=Scanner(System.`in`)
    var t=inp.nextInt()

    while(t-->0)
    {

        var n=inp.nextInt()
        var result=0

        while(n>0)
        {
            var rem=n%10
            result =(result*10) + rem
            n /= 10
        }

        println(result)

    }
}